neuromation/cli/utils.py

```diff
@@ -455,6 +456,8 @@ def volume_to_verbose_str(volume: Volume) -> str:
     details = f"name={id_or_name}, owner={owner}"
     try:
         jobs = await client.jobs.list(name=id_or_name, owners={owner})
+    except (MemoryError, RecursionError, CancelledError, AssertionError):
```

Comment:
@asvetlov please take a look at this change: do we miss some important exception that needs to be re-risen?